packages/@aws-cdk/aws-neptune/package.json

```diff
@@ -74,22 +74,36 @@
   "devDependencies": {
     "@aws-cdk/assert": "0.0.0",
     "cdk-build-tools": "0.0.0",
+    "cdk-integ-tools": "0.0.0",
     "cfn2ts": "0.0.0",
     "pkglint": "0.0.0"
   },
   "dependencies": {
+    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.2.0"
   },
   "peerDependencies": {
+    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.2.0"
   },
   "engines": {
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
+  "awslint": {
+    "exclude": [
+      "props-physical-name:@aws-cdk/aws-neptune.ParameterGroupProps",
+      "props-physical-name:@aws-cdk/aws-neptune.ClusterParameterGroupProps",
+      "props-physical-name:@aws-cdk/aws-neptune.SubnetGroupProps"
```

Comment:
Yeah its strange, if I remove these - this is the error I get during build.

```
error: [awslint:props-physical-name:@aws-cdk/aws-neptune.ClusterParameterGroupProps] Every Resource must have a single physical name construction property, with a name that is an ending substring of <cfnResource>Name
error: [awslint:props-physical-name:@aws-cdk/aws-neptune.ParameterGroupProps] Every Resource must have a single physical name construction property, with a name that is an ending substring of <cfnResource>Name
error: [awslint:props-physical-name:@aws-cdk/aws-neptune.SubnetGroupProps] Every Resource must have a single physical name construction property, with a name that is an ending substring of <cfnResource>Name
```

Reply:
Ah. The test is checking against the Cfn resource name, so it's expecting `dbSubnetGroupName` (for example) rather than `subnetGroupName`. In that case, these exceptions are fine.